lib/manageiq/appliance_console/message_configuration.rb

```diff
@@ -112,14 +112,11 @@ def configure_messaging_yaml
 
         messaging_yaml = YAML.load_file(messaging_yaml_sample_path)
 
-        messaging_yaml["production"].delete("username")
-        messaging_yaml["production"].delete("password")
-
-        messaging_yaml["production"]["hostname"]          = message_server_host
```

Comment:
We were setting the `hostname` key then later in MiqQueue changing it to `host` here https://github.com/ManageIQ/manageiq/pull/22171/files#diff-0e5efff55cd8273572873c343f228135e77fd302231c419486fa601e343af459L693